site/guide/user-management.qmd

```diff
@@ -0,0 +1,66 @@
+---
+title: "User management"
+keywords: "user management, RBAC, roles, groups, permissions"
+---
+
+## Introduction 
+
+User management involves controlling and organizing user accounts within the ValidMind Platform UI. It encompasses tasks like user creation, authentication, authorization, and access control. Effective user management is essential for maintaining security and operational efficiency. 
+
+<!--- A comment --->
+
+## Key concepts
+
+ValidMind uses role-based access control (RBAC) to manage user permissions systematically for our platform. With RBAC, access permissions are associated with roles rather than individual users. Users are assigned to specific roles, and these roles are then granted the necessary permissions to perform their respective tasks. 
+
+This approach simplifies user management by centralizing permissions and enhances security and efficiency. It ensures that your users have the appropriate level of access based on their roles, reducing the risk of unauthorized actions and fostering a structured and controlled environment within an organization's digital ecosystem.
+
+![](user-management.jpg){width="80%"}
+
+### Key terms
+
+Users
+: Individuals interacting with the Platform UI, each having a unique account.
+
+Roles
+: Sets of permissions defining actions that user can perform on resources in the Platform UI that can be taken.
+
+Groups 
+: Collections of users with similar roles or permissions for streamlined management, such as an organization or team within your company. 
```

Comment:
Small comment: we might want to mention that groups allow for grouping at the operational / business unit level instead of a grouping with similar roles or permissions.

Even though groups allow for any arbitrary grouping of users, we expect customers will create groups such as:
- Wealth management
- Retail banking
- etc

And put all their developers and validators in there, versus groups such as:

- Developers
- Validators
- Governance
- etc.

Thoughts @mehdi0501? At the end it's a customer decision but I think we might be steering users towards one side when we say "collection of users with similar roles".

Reply:
I made some changes based on the suggestion from Mehdi to include an example and reworked the text to emphasize the grouping at the operational or business unit level.